### DISCUSSION TITLE
How can we join the chat beta as mentioned on twitter?

### DISCUSSION THREAD

**Discussion by rich-martin** [KnowledgeCanvas/knowledge#104] [Q&A] [answered]
I was wondering what the process is? I’d very much like to join the beta, please.

**Comment by RobRoyce** [comment]
I'll be putting out a form soon, stay tuned!

**Reply by rich-martin** [reply]
Where should I look out for that? 🙂

**Comment by RobRoyce** [comment, marked as answer]
[Beta signup form](https://forms.gle/aJPe7FhwypRbZvRa6) up now! Beta release to follow soon, stable release some time after.

**Reply by rich-martin** [reply]
Done!
